### ISSUE TITLE
Improve documentation for ndex parameter in POST endpoints

### ISSUE THREAD

**Issue by coleslaw481** [coleslaw481/nbgwas_rest#13]
Add http://www.ndexbio.org/#/network/f93f402c-86d4-11e7-a10d-0ac135e8bacf
as example ndex UUID to use to the ndex parameter in POST endpoints. This can either be a default value (might cause problems since only one of the following parameters can be set **network, ndex, column**) OR add it to the description of the parameter.

**Comment by coleslaw481** [comment]
Also update description and parameters

```Bash
This system is designed to use biological networks to analyze GWAS results. A GWAS association score is assigned to the genes. A molecular network is downloaded from the NDEx database, and network propagation is performed, providing a set of new scores for each gene. The top hits on this list form a new subnetwork, which can be compared to a set of gold standard genes in order to evaluate the enrichment for previously discovered biology. See https://github.com/shfong/nbgwas for details.
```

Parameter alpha

```Bash
This model accepts a propagation constant alpha between 0 and 1, representing the probability of walking to network neighbors as opposed to reseting to the original distribution. Larger values induce more spread on the network.
```

**Comment by coleslaw481** [comment]
Fixed with release 0.1.1
